Tools/autotest/arducopter.py

```diff
@@ -4684,14 +4708,25 @@ def loiter_requires_position(self):
         self.context_push()
         self.set_parameter("GPS_TYPE", 0)
```

Comment:
change SIM_GPS_DISABLE=1

Reply:
changed to set SIM_GPS_DISABLE instead of GPS_TYPE